.github/workflows/pr-plugin-build.yml

```
@@ -0,0 +1,154 @@
+  build-plugin:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      
+      - name: Get changed files
+        id: changed-files
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:11:31: Double quote to prevent globbing and word splitting

(shellcheck)